src/blog/2022/11/flowforge-1-1-released.md

```diff
@@ -0,0 +1,62 @@
+---
+title: FlowForge 1.1 released
+subtitle: Node-RED editor on Devices, persistant file storage and import snapshots from outside FlowForge
+description: Node-RED editor on Devices, persistant file storage and import snapshots from outside FlowForge
+date: 2022-11-24 18:00:00.0
+authors: ["rob-marcer"]
+video: TODO
+---
+
+Edit your flows directly on your FlowForge Devices, persist files on your FlowForge projects, and import Node-RED snapshots into FlowForge.
+
+<!--more-->
+
+Keep reading for the details of what's in this release or you can watch our 1 minute roundup video of the new release above. 
+
+We're pleased to announce version 1.1 is now available. The next release of the FlowForge application containing new features, a number of improvements, and bug fixes. Keep reading for a promotion code to get your first month free on FlowForge Cloud. 
+
+
+## Features
+[Node-RED Editor on Devices](https://github.com/flowforge/flowforge/issues/1012) We've added the ability for you to edit flows directly on your Devices. 
+
+[Persistant File Storage](https://github.com/flowforge/flowforge/issues/998) We've completed the first steps to allow files to be created and persisted within your projects. This can be used to store files locally to log data or store current settings for later retrieval. 
+
+[Import Snapshots from Outside FlowForge](https://github.com/flowforge/flowforge/issues/1165) You can now take flows created outside of FlowForge and easily import them into your projects. 
+
+## Improvements
+
+A lot of effort has been put in the install documentation in this release. Most
+notably there's no requirement to setup Mosquitto in the local install method
+and [updated all screenshots](https://github.com/flowforge/flowforge/pull/1189) after the redesign.
```

Comment:
Add a paragraph with the other documentation improvements too. 

Problem: Took long to install FlowForge
When: New users trying to install through LocalFS
Why: We believe that FlowForge offers tremendous value, for free
So: The wizard now installs a default stack and template (Which are advanced topics, now admins can deal with them later)

Reply:
Problem: Not sure how to configure Mosquitto
When: Installing FF with LocalFS
Why: Devices update faster with MQTT
So: A default conf file is provided

See also: https://github.com/flowforge/installer/issues/50